docker/dev/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.14
+FROM golang:latest
```

Comment:
**[hadolint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
DL3007 Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

Reply:
Asshole!!!